src/components/more-blogs-section/more-blogs-section.jsx

```diff
@@ -10,15 +10,15 @@ import { Card } from "../card";
 import * as styles from './more-blogs-section.module.scss';
 
 const MoreBlogsSection = ({ data, refMoreFrom, title }) => {
-  const { edges } = data.allMdx;
+  const { relatedPosts } = data.mdx;
 
   return (
     <div className={styles.moreFromXlContainer} ref={refMoreFrom} >
       <div className={styles.titleContainer}>
-        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>{`${title} →`}</Link>
+        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>More from about <span>{title}</span>→</Link>
```

Comment:
Why this? I prefer:
```suggestion
        <Link to="/" className={classnames("text__heading__three__blueTwo", styles.titleStyle)}>{`More from about ${title} →`}</Link>
```

Reply:
Because I need to add another color to the title, and only to the title.